backend/src/api/player_view_functions.clj

```diff
@@ -2,22 +2,31 @@
   (:require [rules.victory-conditions :as victory]
             [configs.messages :as messages]))
 
+(defn ^:private partial-card
+  "Returns only certain keys of a card and change the :owner"
+  [card wanted-keys owner]
+  (assoc (select-keys card wanted-keys)
+         :owner owner))
+
 (defn get-cards
   "Returns cards as seen by a player"
   [game-state player-id]
   (vec (map
          #(cond
-           (= (:owner %) player-id)
-           (assoc % :owner "me")
+            (and (and (= (:owner %) player-id)
```

Comment:
double `and` is redundant. In clojure `and` takes any amount of arguments, so you can just do `(and cond1 cond2 cond3 ... condN)`